### PR TITLE
Additional testing for pax installs

### PR DESCRIPTION
**Pull request by DevonianTeuchter**
Tests the initial drop of pax install

### CODE REVIEW

File: tests/zopen_check_paxinstall [hunk 1]
```diff
@@ -0,0 +1,154 @@
+#!/bin/sh
+
+#
+# FVT of zopen
+#
+#set -e # hard failure if any commands fail
+set -x
+WORKDIR="$1"
+
+fail(){ echo "$@"; exit 8; }
```
Comment (IgorTodorovskiIBM) [inline tests/zopen_check_paxinstall:10]: I like this, perhaps we can make these functions available in a common script for consistency (could be in another PR)

File: tests/zopen_check_paxinstall [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+#trap -- "rm -rf ${zopenenv}" EXIT INT TERM QUIT HUP
+mkdir -p "${zopenenv}"
+zopen_tool_binary=$(whence zopen)
+zopen init -y --re-init "${zopenenv}"  #specify re-init to ensure an env
+echo "Rc=$?"
+
+echo "Testing if zopen was installed at: ${zopenenv}/usr/local/zopen"
+[ ! -e "${zopenenv}/usr/local/zopen" ] && fail "File system not available"
+echo "Testing source of the zopen-config file"
+[ ! -e "${zopenenv}/etc/zopen-config" ] && fail "xopen configuration not available"
```
Comment (IgorTodorovskiIBM) [inline tests/zopen_check_paxinstall:24]:
```suggestion
[ ! -e "${zopenenv}/etc/zopen-config" ] && fail "zopen configuration not available" 
```

File: tests/zopen_check_paxinstall [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+echo "Rc=$?"
+
+echo "Testing if zopen was installed at: ${zopenenv}/usr/local/zopen"
+[ ! -e "${zopenenv}/usr/local/zopen" ] && fail "File system not available"
+echo "Testing source of the zopen-config file"
+[ ! -e "${zopenenv}/etc/zopen-config" ] && fail "xopen configuration not available" 
+
+echo "Testing source of configuration file"
+# shellcheck disable=SC1091
+. "${zopenenv}/etc/zopen-config"
```
Comment (IgorTodorovskiIBM) [inline tests/zopen_check_paxinstall:28]: Does it make sense to check the return code as well?

**Review by IgorTodorovskiIBM — APPROVED** [review]
Looks good other than a few minor comments, great job!

**Comment by DevonianTeuchter** [comment]
Closing - the new install process looks different to what this PR was originally going to test!
